app/models/bgs_dependents/child_school.rb

```diff
@@ -7,8 +7,7 @@ class ChildSchool < Base
     attribute :program_information, Hash
     attribute :current_term_dates, Hash
 
-    def initialize(dependents_application, proc_participant)
```

Comment:
Is the proc_participant no longer needed? Just trying to understand what's going on here

Reply:
We've been doing a lot of refactoring due to breaking up a larger PR into multiple smaller PRs.  This functionality has been moved to `/lib/bgs/student_school.rb` in the `submit_params` method.

Reply:
Not to make it even more confusing, but due to some more refactoring and feedback suggestions, we've added this back in the `params_for_686c` method.